Library/Formula/bazaar.rb

```diff
@@ -20,11 +20,8 @@ def install
     system "make", "man1/bzr.1"
     man1.install "man1/bzr.1"
 
-    # Put system Python first in path
-    ENV.prepend_path "PATH", "/System/Library/Frameworks/Python.framework/Versions/Current/bin"
-
     system "make"
-    inreplace "bzr", "#! /usr/bin/env python", "#!/usr/bin/python"
+    inreplace "bzr", "#! /usr/bin/env python", "#!#{which "python"}"
```

Comment:
My preference is to trust the user's `PATH`. As a user if I upgrade my Python distribution for example with `brew install python`, I'd prefer that installed packages started using it.